Gulpfile.js

```diff
@@ -19,7 +20,29 @@ const banner = `/**
  */
 `;
 
-gulp.task('lint', () => {
+gulp.task('lint:ec', () => {
+    const sources = [
```

Comment:
Because we have to check dotfiles too, and ignore built ones and `node_modules`, it is more convenient to whitelist than blacklist.